tests/agents/test_agent.py

```
@@ -366,6 +366,34 @@ def test_tools_initialized(self, mock_create_agent):
+        mock_agent_executor.invoke.side_effect = error
+
+        mock_parsing_llm = Mock(spec=BaseChatModel)
+        agent = CodeBoardingAgent(
+            repo_dir=self.repo_dir,
+            static_analysis=self.mock_analysis,
+            system_message="Test",
+            agent_llm=self.mock_llm,
+            parsing_llm=mock_parsing_llm,
+        )
+
+        with self.assertRaises(Exception, msg="model not found"):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)